jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshTemplate.java

```diff
@@ -33,7 +33,7 @@
     private SSHClient sshClient;
 
     private abstract static class Commands {
-        static final String TIMEZONE = "timedatectl | grep zone | grep -Po '(.*zone: )\\K(\\S+)'";
+        static final String TIMEZONE = "date +%z";
```

Comment:
The output will be different but the `ZoneId.of()` is capable of parsing multiple input variants - so it will still return the same ZoneId as before even though the input string representation of zone (name vs offset) is different.